src/icrar/leap-accelerate-cli/Arguments.h

```diff
@@ -114,6 +114,9 @@ namespace icrar
          */
         std::unique_ptr<MeasurementSet> m_measurementSet;
         std::istream* m_inputStream = nullptr; // Cached reference to the input stream
+        
+        std::unique_ptr<std::ofstream> m_outputFileStream;
+        std::ostream* m_outputStream = nullptr;
```

Comment:
`m_outputStream` doesn't need to be stored separately, it's already contained in the `unique_ptr`